src/ziggurat/messaging/producer.clj

```diff
@@ -185,7 +214,9 @@
     (make-channel-queue topic-entity channel :instant)
     (when (channel-retries-enabled topic-entity channel)
       (make-channel-delay-queue topic-entity channel)
```

Comment:
why are we making an extra delay queue when retry and exponential backoff both are enabled?

Reply:
updated, remove delay queue creation when exponential backoff is enabled